src/views/components/create-model-form.jsx

```diff
@@ -73,6 +75,9 @@ export default class CreateModelForm extends React.Component {
             this.state.price,
             this.state.count
         );
+        Dispatcher.handleAction('CREATE_TOAST', {
```

Comment:
This should go in the `then` portion of the `createModel` method. If that were to fail for any reason this would give a false positive.